internal/apiutil/errors.go

```diff
@@ -101,4 +98,13 @@ var (
 
 	// ErrBootstrapState indicates an invalid boostrap state.
 	ErrBootstrapState = errors.New("invalid bootstrap state")
+
```

Comment:
It's not related to this PR, but check for the typos in this file (`malmormed`, `boostrap`)